css/build/style.css

```diff
@@ -199,8 +263,148 @@ p {
 }
 
 /* body */
-.content-1 {
-  min-height: 30vh;
+.body-content-1 {
+  display: grid;
+  width: calc(100% - 48px);
+  padding: 114px 24px;
+  gap: 88px;
+}
+
+.works-card {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  gap: 12px;
+  padding: 16px;
+  background-color: white;
+  border-style: solid;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.work-card-overline ul {
+  display: flex;
+  list-style-type: none;
+  justify-content: flex-start;
+  align-content: center;
+  align-items: center;
+  padding: 0;
+  height: 0;
+  gap: 10px;
+}
+
+.work-card-techstack {
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.work-card-techstack ul {
+  display: flex;
+  list-style-type: none;
+  justify-content: flex-start;
+  align-content: center;
+  align-items: center;
+  padding: 0;
+  height: 0;
+  gap: 10px;
+}
+
+.work-card-techstack li {
+  padding: 4px 12px;
+  background: #ebebff;
+  border-radius: 8px;
+}
+
+.work-card-button {
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  padding: 12px;
+  margin: 12px 0;
+  transition-duration: 0.4s;
+}
+
+.work-card-button:hover {
+  background-color: #6070ff;
+  color: white;
+}
```

Comment:
Great job implementing the hover feature, really smooth transition, looks great! 🎉 
However, you're still missing the `active` interaction. Please add it.

<img width="833" alt="Screen Shot 2022-05-26 at 14 39 29" src="https://user-images.githubusercontent.com/82426682/170564505-ed867852-1e4e-4dd3-a542-2cf6da5ae9bd.png">